ImperatorToCK3Tests/ImperatorWorldTests/Countries/CountryTests.cpp

```diff
@@ -129,4 +129,110 @@ TEST(ImperatorWorld_CountryTests, currenciesCanBeSet)
 	ASSERT_EQ(50, theCountry.getCurrencies().aggressive_expansion);
 	ASSERT_EQ(4, theCountry.getCurrencies().political_influence);
 	ASSERT_EQ(1, theCountry.getCurrencies().military_experience);
+}
+
+TEST(ImperatorWorld_CountryTests, monarchCanBeSet)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "\tmonarch=69";
+	input << "}";
+
+	const ImperatorWorld::Country theCountry(input, 42);
+
+	ASSERT_EQ(69, theCountry.getMonarch());
+}
+
+TEST(ImperatorWorld_CountryTests, monarchDefaultsToMinusOne)
+{
+	std::stringstream input;
+	input << "=\n";
+	input << "{\n";
+	input << "}";
+
+	const ImperatorWorld::Country theCountry(input, 42);
+
+	ASSERT_EQ(-1, theCountry.getMonarch());
```

Comment:
I guess this was set in a previous change, but using a std::optional would be better than using a sentinel value.

Reply:
In a previous PR @Zemurin advised the opposite for primitives. I really think this should be decided upon on organization level to avoid further confusion.

Reply:
Changed to optional ✔️